### ISSUE TITLE
"_" in project_slug breaks vagrant with virtualbox

### ISSUE THREAD

**Issue by pacejackson** [reddit/baseplate-cookiecutter#4]
If you have an "_" character in your project_slug, it will create an invalid vagrantfile.

```
➜  cass_test vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* The hostname set for the VM should only contain letters, numbers,
hyphens or dots. It cannot start with a hyphen or dot.
```

Line 5 seems to be the offending line: 
```config.vm.hostname = "cass_test.local"```

**Comment by pacejackson** [comment]
This was on OSX using VirtualBox version 5.0.20 r106931

**Comment by spladug** [comment]
Fixed in 56bb7caf26c799838b599d32e4eb47ca464fbc62
